web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -277,7 +280,11 @@
                         { locale: $locale },
                       )}
                     >
-                      Age {age}
+                      {#if age == 0 && ageInMonths >= 0 && ageInMonths <= 11}
```

Comment:
You've got some redundant conditions there: if age is 0 ageInMonths will always be <= 11 (per definition). Also, I think it's save to assume that ageInMonths won't be negative (given that we didn't check this atm either).

Reply:
Fair points, I simplified the conditions.